index.js

```diff
@@ -1,156 +1,156 @@
 'use strict';
 
-module.exports = Plugin;
-function Plugin(inputNodes, options) {
-  if (!(this instanceof Plugin)) throw new TypeError('Missing `new` operator');
-  // Remember current call stack (minus "Error" line)
-  this._instantiationStack = new Error().stack.replace(/[^\n]*\n/, '');
-
-  options = options || {};
-  if (options.name != null) {
-    this._name = options.name;
-  } else if (this.constructor && this.constructor.name != null) {
-    this._name = this.constructor.name;
-  } else {
-    this._name = 'Plugin';
-  }
-  this._annotation = options.annotation;
-
-  let label = this._name + (this._annotation != null ? ' (' + this._annotation + ')' : '');
-  if (!Array.isArray(inputNodes))
-    throw new TypeError(
-      label + ': Expected an array of input nodes (input trees), got ' + inputNodes
-    );
-  for (let i = 0; i < inputNodes.length; i++) {
-    if (!isPossibleNode(inputNodes[i])) {
+class Plugin {
+  constructor(inputNodes, options) {
+    if (!(this instanceof Plugin)) throw new TypeError('Missing `new` operator');
```

Comment:
this line is not longer needed, es6 class constructors cannot be invoked without new.

Reply:
Ya I was thinking the same thing and just removed it.